components/admin_real/desktop/DeleteReservationButton.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import { Delete } from "@mui/icons-material";
+import { Button } from "@mui/material";
+import Modal from "antd/lib/modal/Modal";
+import { deleteReservation } from "../shared/reservationTableUtils";
+const DeleteReservationButton = ({ reservationId }) => {
+  const [modalOpen, setModalOpen] = useState(false);
+  return (
+    <div>
+      <Modal
+        title={"Are you sure you want to delete this reservation?"}
+        style={{}}
+        visible={modalOpen}
+        onOk={() => {
+          deleteReservation(reservationId);
+          setModalOpen(false);
+        }}
+        onCancel={() => setModalOpen(false)}
+      >
+        <div>Once you delete it, we won't be able to recover it.</div>
```

Comment:
This has a problem with the deploy:
<img width="1533" alt="image" src="https://user-images.githubusercontent.com/11667804/195150792-4ed6c8d0-b201-4cde-a825-308ffd825def.png">